docs/source/_templates/home.html

```diff
@@ -437,63 +488,78 @@ <h3 class="h-card__title">Mathematics</h3>
-                                    </div>
+                        <h6 class="tabs-info txt-center">With FURY, it becomes simple to produce multiple physics animations with ease.</h6>
+                        <div class="visualization-images">
+                            <div class="image-grid grid g-gap-2 g-col-3 g-row-2 g-m-col-2 g-m-row-3">
+                                <div>
+                                    <a href="fury-pybullet.html"><img
+                                            src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_domino.gif"
+                                            alt="physics domino" align="middle"></a>
+                                            
+                                </div>
+                                <div>
+                                    <a href="fury-pybullet.html"><img
+                                            src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_bricks_multi_actor.gif"
+                                            alt="physics bricks" align="middle"></a>
+                                            
+                                </div>
+                                <div>
+                                    <a href="fury-pybullet.html"><img
+                                            src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_collision.gif"
+                                            alt="physic collision" align="middle"></a>
+                                            
+                                </div>
+                                <div>
+                                    <a href="auto_examples/index.html#demos"><img
+                                            src="https://github.com/fury-gl/fury-communication-assets/raw/main/brownian_motion.gif"
+                                            alt="brownian motion" align="middle"></a>
+                                            
+                                </div>
+                                <div>
+                                    <a href="auto_examples/index.html#demos"><img
+                                            src="https://github.com/fury-gl/fury-communication-assets/raw/main/em_wave.gif" alt="Em wave"
+                                            align="middle"></a>
+                                            
+                                </div>
+                                <div>
+                                    <a href="auto_examples/index.html#demos"><img
+                                            src="https://github.com/fury-gl/fury-communication-assets/raw/main/helical_path.gif"
+                                            alt="Helical path" align="middle"></a>
+                                            
                                 </div>
```

Comment:
Why so few links are using `raw.githubusercontent` and few directly linking to the `fury-communication-assets`?
Can't we use consistent links?

Also please have a look at Github permalinks it would help here!

Reply:
I did not change any link from the existing code. If not I might need to see what to change.

Reply:
Also, using the raw GitHub link would be a better option as the fury-communications-asset is meant for storing the assets.

We don't want to go around and create a permalink of the objects as they are hard to manage.

Reply:
Yup, I also think raw GitHub links might help.
Let's see what @skoudoro has to say!

Reply:
no need for permalink, we keep permalink only for code for now.

Otherwise, I am ok, for both version, they point to the same place.